scipy/stats/_stats_py.py

```diff
@@ -5454,11 +5454,11 @@ def spearmanr(a, b=None, axis=0, nan_policy='propagate',
         return res
 
 
-@xp_capabilities(np_only=True)
+@xp_capabilities(cpu_only=True, exceptions=['cupy', 'jax.numpy'])
 @_axis_nan_policy_factory(_pack_CorrelationResult, n_samples=2,
                           result_to_tuple=_unpack_CorrelationResult, paired=True,
                           too_small=1, n_outputs=3)
-def pointbiserialr(x, y):
+def pointbiserialr(x, y, *, axis=0):
```

Comment:
Running the full array API testsuite for `stats` passes just fine on this branch without the `axis` shim. A simple enough change that no more testing is worth adding for that?

Reply:
It does get tested - the `_axis_nan_policy` decorator sees that `pointbiserialr` accepts `axis`, and so it passes it `axis` rather than assuming it is a 1-D implementation that needs to be wrapped. And the `_axis_nan_policy` tests are already testing the `axis` argument of the public function.